### DISCUSSION TITLE
brew fontforge

### DISCUSSION THREAD

**Discussion by davidescicolone** [Homebrew/discussions#934] [Everyday usage]
Dear all,

I'm trying to install the package fontforge through brew but if I run the command `brew install fontforge` I receive the following error message:

...
Error: fontforge: no bottle available!
You can try to install from source with:
  brew install --build-from-source fontforge
...

the same command works to my colleagues.
I'm running it in my Mac Book Pro (with processor Apple Silicon M1 (ARM based)) with BigSur OS.
My colleagues run it on Mac Book Pro with Intel-based processor and BigSur OS as well.

The processor could be the reason? does brew search the packages also based on the processor family (e.g. ARM vs Intel)? 
If so, how can force to download the  intel based version?

Thank you!

Davide

**Comment by SMillerDev** [comment]
> The processor could be the reason? does brew search the packages also based on the processor family (e.g. ARM vs Intel)?

Yes it does, since binaries for x86 are very different from binaries for ARM.

> If so, how can force to download the intel based version?

Install Homebrew with Rosetta, see for example: https://github.com/Homebrew/discussions/discussions/885
